cumulusci.yml

```diff
@@ -290,18 +299,26 @@ flows:
                 options:
                     unmanaged: False
 
-    install_regression:
+    config_regression:
         steps:
-            # Add QA metadata needed for regression
-            5:
+            1:
                 task: deploy_qa_config
                 options:
                     unmanaged: False
```

Comment:
The table you're working on made me realize that this refactoring lost the `config_managed` flow here (which was the last step of install_regression in CumulusCI). The automated tests used for regression need the full configuration applied by config_managed, plus the additional metadata added by deploy_qa_config.

Reply:
Sorry, I'm confused. Here's what `cci flow info install_regression` says on my machine:

```yaml
description: Install the latest beta dependencies and upgrade to the latest beta version from the most recent production version
steps:
    1:
        flow: beta_dependencies
    2:
        task: install_managed
        options:
            version: previous
    3:
        task: install_managed_beta
    4:
        flow: config_managed
```

Are we moving `config_managed` out of `install_regression` in CumulusCI?

Reply:
Ah right. I have proposed that change in https://github.com/SFDO-Tooling/CumulusCI/pull/1088, but was forgetting that this was written before that.